XIPs/xip-52-group-announcement.md

```diff
@@ -31,7 +31,9 @@ This would be the similar to a description of a group, but each app can display
 
 ## Backward compatibility
 
-This is a breaking change. To use this new `announcement` metadata field, all group members must be using an app that uses an SDK version that supports the metadata.
+If a group exists before we add group announcements. then it will have no value for group announcement in mutable metadata and no permission policy for who is allowed to update a mutable metadata field with the "groupAnnouncement" key.
+Then someone on the new version calls updateGroupAnnouncement on that group to set that value, even though there is no existing permission policy in that group because it is already created. In that case we have default permission policies to evaluate if the metadata update is valid or not.
+If someone on an old version is invited to a group from someone on the new version, and so there is a permission policy set for groupAnnouncement.  Users on the older versions will actually be able to read the permission policy because it is fetched using that string key.
```

Comment:
How about something like:

```suggestion
- If a user on an older version of the protocol is invited to a group created by a user on the newer version, then the group will have a permission policy set for `groupAnnouncement`. Even though the older version of the protocol doesn't support setting announcements, it can still read the permission policy because it's stored as a key-value pair in the metadata.
```